DNN Platform/Library/Entities/Users/UserRegistrationEmailNotifier.cs

```diff
@@ -0,0 +1,59 @@
+﻿using DotNetNuke.Common;
+using DotNetNuke.Common.Utilities;
+using DotNetNuke.Entities.Portals;
+using DotNetNuke.Services.Mail;
+using static DotNetNuke.Common.Globals;
+
+namespace DotNetNuke.Entities.Users
+{
+    public class UserRegistrationEmailNotifier
+    {
+        private static PortalSettings __portalSettings = PortalSettings.Current;
```

Comment:
Having this field be `static` will cause it to never be updated, correct?  It seems like we should just use `PortalSettings.Current` throughout, and/or require a `PortalSettings` instance in the constructor.

Reply:
great observation.. fixed now!